source/includes/azure/_instances.md

```diff
@@ -219,21 +218,20 @@ curl -X POST \
 Update the machine type for the existing instance. 
 
 <aside class="notice">
-Each machine type has a predefined number or vCPU, memory and disk limitation and capacity.
-Click on th link to get see the list is machine type detailed for <a href="https://docs.microsoft.com/en-us/azure/virtual-machines/windows/sizes">Windows</a>
+Each machine type has a predefined number of vCPUs, memory, and disk limitation and capacity.
```

Comment:
Do we need to remove the extra and?

Reply:
I think this is okay, since the limitation and capacity is on the disk size, but would also replace "number" by "amount"

Reply:
i find amount is more related to money